docs/src/main/paradox/api-reference/acls.md

```diff
@@ -0,0 +1,138 @@
+can be subsequently modified by users belonging to administrator groups.
+
+**Note**: Be careful when modifying top-level permissions, as it can results in locking everyone out, or
+on the contrary, giving full ownership to everyone, including anonymous users.  No checks about the sanity of
+such operations are performed by the service.
+
+## Actions
+
+### Fetch own permissions on a resource
+
+```
+GET /v0/acls/{address}
+```
+
+**Note**: Permissions are transitively inherited from the resource ACL and all its ancestors.
+This call computes the effective ACL for the caller on this resource.
+
+#### Status Codes
+
+- **200 OK**: the ACL is computed and the associated caller permissions are returned successfully
+- **403 Forbidden**: the caller doesn't have `read` rights on this resource
+
+## Administrative actions
+
+### Fetch permissions for all users on a resource
+
+```
+GET /v0/acls/{address}?all=true
+```
+
+**Note**: This action *does not* show inherited permissions.
+
+#### Status Codes
+
+- **200 OK**: the entire ACL is fetched is returned successfully
+- **403 Forbidden**: the caller doesn't have `own` rights on this resource
+
+### Create a new ACL
+
+```
+PUT /v0/acls/{address}
+{...}
+```
+
+Payload
+:   @@snip [acls.json](../assets/api-reference/acls.json)
+
+#### Status Codes
+
+- **201 Created**: the resource ACL was created successfully
+- **400 Bad Request**: the request payload is not valid
+- **403 Forbidden**: the caller doesn't have `own` rights on this resource
+- **409 Conflict**: the resource ACL already exists
+
+### Add permissions to a resource ACL
+
+```
+POST /v0/acls/{address}
+{...}
+```
+
+Payload
+:   @@snip [acl.json](../assets/api-reference/acl.json)
+
+#### Status Codes
+
+- **200 OK**: the resource ACL was updated successfully
+- **400 Bad Request**: the request payload is not valid
+- **403 Forbidden**: the caller doesn't have `own` rights on this resource
+
+### Clear ACL on a resource
+
+```
+DELETE /v0/acls/{address}
+```
+
+#### Status Codes
+
+- **200 OK**: the resource ACL was cleared successfully
+- **403 Forbidden**: the caller doesn't have `own` rights on this resource
+- **404 Not Found**: the resource ACL was not found, i.e. it is already empty
+
+## Permission format
+
+The `permissions` object in the payload (in both request and response body) is a simple
+JSON array of arbitrary string literals. Internally, the IAM services
+recognizes `read`, `write` and `own`. Third party services can store and
+make use of custom permissions.
+
+## Identity format
+
+The `identity` object in the payload (in both request and response body) is qualified by
+a mandatory `type` field that needs to match one of the literals described below. Additionally,
+any authenticated identity has an `origin` which is the URI of the provider realm. Groups
+have a `group` identifier, and users a `name`.
+
+| Type | Additional fields | Description |
```

Comment:
Could we give out some examples here? or just replace the table with tabbed snips of examples (with comments)?